### PR TITLE
ER-1204 - Update slack message notifications

### PR DESCRIPTION
**Pull request by sachpatel**

### CODE REVIEW

File: src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "VacancyAnalyticEventsSqlDbConnectionString": "Data Source=127.0.0.1,1401;Initial Catalog=das-vacancy-analytic-events;User Id=sa;Password=Ozzyscottypassword1;MultipleActiveResultSets=True"
   },
   "Slack": {
-    "WebHookUrl": ""
+    "FindAnApprenticeshipDetailPrefixUrl": "https://www.findapprenticeship.service.gov.uk/apprenticeship/"
```
Comment (chrisjensenuk) [inline src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json]: Wonder if this should just contain the domain name only `https://www.findapprenticeship.service.gov.uk`. Then the notifier can append the path as needed.  Then we could redirect to different locations in FAA for different types of notifications without needed extra config. But there is no demand for this yet so this is fine. Just a musing.

File: src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj [hunk 2]
```diff
@@ -20,7 +20,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="CsvHelper" Version="12.1.2" />
-    <PackageReference Include="HtmlSanitizer" Version="4.0.205" />
+    <PackageReference Include="HtmlSanitizer" Version="4.0.210" />
```
Comment (chrisjensenuk) [inline src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj]: QA should explicitly regression test all these package updates

File: src/Employer/Employer.Web/Employer.Web.csproj [hunk 3]
```diff
@@ -9,7 +9,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="FluentValidation.AspNetCore" Version="7.5.2" />
-    <PackageReference Include="Humanizer.Core.uk" Version="2.5.16" />
+    <PackageReference Include="Humanizer.Core.uk" Version="2.6.2" />
```
[the remark this answers is not in the document]
Comment (sachpatel) [reply src/Employer/Employer.Web/Employer.Web.csproj]: yes

File: src/Shared/Recruit.Vacancies.Client/Application/EventHandlers/NotifyOnVacancyActionsEventHandler.cs [hunk 4]
```diff
@@ -9,64 +9,35 @@
...
-        }
-
-        public async Task Handle(VacancyReviewCreatedEvent notification, CancellationToken cancellationToken)
-        {
-            try
-            {
-                var vacancyReview = await _vacancyReviewRepository.GetAsync(notification.ReviewId);
-
-                await _notifier.VacancyReviewCreated(vacancyReview);
+                if (vacancy.ClosedByUser != null)
```
[the remark this answers is not in the document]
Comment (sachpatel) [reply src/Shared/Recruit.Vacancies.Client/Application/EventHandlers/NotifyOnVacancyActionsEventHandler.cs]: Have put a status check in.

File: src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json (hunk 1 above)
Comment (sachpatel) [reply src/Jobs/Recruit.Vacancies.Jobs/appSettings.Development.json]: This is for purpose so it is fine.

File: src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj (hunk 2 above)
Comment (sachpatel) [reply src/Shared/Recruit.Vacancies.Client/Recruit.Vacancies.Client.csproj]: Yes

**Pull request merged**